### DISCUSSION TITLE
How to replace eval("$A.get('e.force:refreshView').fire();"); in LWC when LWS is enabled?

### DISCUSSION THREAD

**Discussion by ankit-advologix** [eslint/eslint#18553] [Q&A]
Having eval("$A.get('e.force:refreshView').fire();"); in code gives error - eval can be harmful.

**Comment by Rec0iL99** [comment]
Hi @ankit-advologix, could you please give us more info on what you are looking for? Is this related to ESLint?

**Reply by ankit-advologix** [reply]
Hi @Rec0iL99 
Yes, ESLint gave this error in Salesforce - Lightning Web Component that eval can be harmful.

**Reply by Rec0iL99** [reply]
We don't maintain the Salesforce ESLint plugin. Please contact the maintainers there for more help.
